marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -147,7 +153,7 @@ private void initializeFlow(String jobId) {
         jobStoppedOnError.set(false);
         runningJobId = jobId;
         runningFlow = flowMap.get(runningJobId);
-        if(jobDocManager == null) {
+        if(jobDocManager == null && !disableJobOutput) {
```

Comment:
I think we need tests to verify that a) no job data is created, and b) nothing blows up when job data is disabled.

Reply:
Added a test for this in the flow runner tests.